### PR TITLE
Niad 1643 change patient's gender mapper

### PR DESCRIPTION
**Pull request by ssokolowskisebastian**

### CODE REVIEW

File: service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/GenderMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hl7.fhir.dstu3.model.Enumerations.AdministrativeGender.FEMALE;
+import static org.hl7.fhir.dstu3.model.Enumerations.AdministrativeGender.MALE;
+import static org.hl7.fhir.dstu3.model.Enumerations.AdministrativeGender.OTHER;
+import static org.hl7.fhir.dstu3.model.Enumerations.AdministrativeGender.UNKNOWN;
+
+import org.junit.jupiter.api.Test;
+
```
Comment (mrogowska) [inline service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/GenderMapperTest.java:10]: Unnecessary empty lines

**Review by mrogowska — CHANGES_REQUESTED** [review]

File: service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/GroupMapperTest.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+
```
Comment (dawidskainos) [inline service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/GroupMapperTest.java:11]: should be removed

File: service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/GenderMapperTest.java (hunk 1 above)
Comment (ssokolowskisebastian) [reply service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/GenderMapperTest.java:10]: done

File: service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/GroupMapperTest.java (hunk 2 above)
Comment (ssokolowskisebastian) [reply service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/GroupMapperTest.java]: done

**Review by mrogowska — APPROVED** [review]

**Pull request merged**